Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:20.04
+
+ARG IMAGE_CREATION
+ARG IMAGE_VERSION
+
+LABEL fr.cyberry.image.created=${IMAGE_CREATION}
+LABEL fr.cyberry.image.authors="Cyberry Team <team@cyberry.fr> (@CyberryTeam)"
+LABEL fr.cyberry.image.url="https://hub.docker.com/r/cyberryteam/epitech-docker"
+LABEL fr.cyberry.image.source="https://github.com/CyberryTeam/epitech-docker"
+LABEL fr.cyberry.image.version=${IMAGE_VERSION}
+LABEL fr.cyberry.image.vendor="Cyberry"
+LABEL fr.cyberry.image.licenses="MIT"
+LABEL fr.cyberry.image.title="Epitech Docker"
+LABEL fr.cyberry.image.description="Docker image for continuous integration and deployment of Epitech projects"
```

Comment:
Adding labels one by one adds layers and in our case it's useless.
Try to minimize the number of instructions in order to limit the number of layers and thus the weight of the image. 
(e.g. : `LABEL fr.cyberry.image.created=${IMAGE_CREATION} fr.cyberry.image.authors="Cyberry Team <team@cyberry.fr> (@CyberryTeam)"` etc.)

Reply:
> Only the instructions RUN, COPY, ADD create layers. Other instructions create temporary intermediate images, and do not increase the size of the build.

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers :grin:

Reply:
I hadn't seen that since 1.10

Reply:
By the way, the labels comply with the standards of the Open Containers Initiative (https://github.com/opencontainers/image-spec/blob/master/annotations.md).